torch/csrc/functorch/init.cpp

```diff
@@ -0,0 +1,380 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+// All rights reserved.
+//
+// This source code is licensed under the BSD-style license found in the
+// LICENSE file in the root directory of this source tree.
+
+#include <torch/extension.h>
+#include <ATen/WrapDimUtils.h>
+#include <ATen/FunctionalTensorWrapper.h>
+
+#include <ATen/functorch/TensorWrapper.h>
+#include <ATen/functorch/DynamicLayer.h>
+#include <ATen/functorch/BatchedTensorImpl.h>
+#include <ATen/functorch/LegacyVmapTransforms.h>
+#include <ATen/functorch/BatchedFallback.h>
+#include <ATen/functorch/BatchRulesHelper.h>
+#include <ATen/functorch/PlumbingHelper.h>
+#include <torch/csrc/functorch/CompileCache.h>
+#include <c10/core/AutogradState.h>
+#include <functorch/csrc/dim/dim.h>
+
+// This file contains functorch's Python bindings.
+
```

Comment:
All of this code was moved from functorch/csrc/init.cpp. The only thing that changed were:
- namespace changes
- I deleted a (now obsolete) `_movedim` helper function.